config-examples/peer-a-config.toml

```diff
@@ -0,0 +1,18 @@
+public_key = "peer-a-public-key"
```

Comment:
Can we come up with a naming scheme that aligns with the whitepaper?

Reply:
you mean `spki`, `sski`, `spkr`, `sskr`? I'm mostly complient, just that I would like to remove the `i`/`r` suffix because you also asked for a more than two peers setup. So maybe `peer-[a..c]-ssk` and `peer-[a..c]-spk` and `peer-[a..c]-osk`?

Reply:
We use `sskm` for `static-secret-key-mine` and `static-secret-key-theirs`. If we use `peer-.*` and `mine-.*` we would need to adjust the whitepaper for compatibility.

Reply:
Though honestly, I am a fan of establishing the four character naming scheme and just using it very consistently. I know its not super intuitive on first use but we need keys in so many places…

Reply:
In general I agree. However, `theirs` and `mine` only works well if there is exactly two peers in total, and its a relative naming which would not work well when both peers are executed on the same host, as for example a mini-demo.